### PR TITLE
Optimization of negate and difference (mathop.c)

### PR DESCRIPTION
**Pull request by bitbank2**

### CODE REVIEW

File: src/omv/img/mathop.c [hunk 1]
```diff
@@ -258,8 +268,8 @@ static void imlib_add_line_op(image_t *img, int line, void *other, void *data, b
                 if ((!mask) || image_get_mask_pixel(mask, i, line)) {
                     int dataPixel = IMAGE_GET_BINARY_PIXEL_FAST(data, i);
                     int otherPixel = IMAGE_GET_BINARY_PIXEL_FAST(((uint32_t *) other), i);
-                    int p = dataPixel + otherPixel;
-                    p = IM_MIN(p, COLOR_BINARY_MAX);
+                    int p = dataPixel | otherPixel; //dataPixel + otherPixel;
```
Comment (kwagyeman) [inline src/omv/img/mathop.c:271]: Remove comments.

Comment (kwagyeman) [reply src/omv/img/mathop.c:271]: For GRAYSCALE and RGB565 you can use the specialized ARM instructions to go faster.

**Comment by iabdalkader** [comment]
This PR has old commits.

File: src/omv/img/mathop.c [hunk 2]
```diff
@@ -674,7 +684,7 @@ static void imlib_difference_line_op(image_t *img, int line, void *other, void *
                 if ((!mask) || image_get_mask_pixel(mask, i, line)) {
                     int dataPixel = IMAGE_GET_BINARY_PIXEL_FAST(data, i);
                     int otherPixel = IMAGE_GET_BINARY_PIXEL_FAST(((uint32_t *) other), i);
-                    int p = abs(dataPixel - otherPixel);
+                    int p = dataPixel ^ otherPixel; // abs(dataPixel - otherPixel);
```
Comment (kwagyeman) [inline src/omv/img/mathop.c:687]: Are there faster ops for the GRAYSACLE and RGB565 paths?

**Pull request closed without merging**